src/biogeochem/CNVegCarbonFluxType.F90

```diff
@@ -4548,10 +4721,35 @@ subroutine Summary_carbonflux(this, &
             this%hrv_deadcrootc_to_litter_patch(p)         + &
             this%hrv_deadcrootc_storage_to_litter_patch(p) + &
             this%hrv_deadcrootc_xfer_to_litter_patch(p)    + &
-            this%hrv_xsmrpool_to_atm_patch(p)              + &
```

Comment:
@lawrencepj1  this is the one thing that's answer changing in this PR.  So I wanted to verify with you that this is correct? And if so I think we should make an issue for it so we've documented what's going on in this change.

Reply:
Hi @ekluzek

Thanks for this. This line comes directly from the tag I checked out: 

Tag name:  ctsm1.0.dev055
Originator(s):  sacks (Bill Sacks)
Date: Tue Aug  6 14:06:50 MDT 2019

This line ensure that when harvest occurs carbon is taken from the xsmr pool (which is negative) and passed by a negative flux to the atmosphere. It have been removed since this tag. My suggestion is that this line should not be added back in rather kept as it is in the current tag.

I can go through all of this code next week to make sure. I will also look through all the gulu new code to make sure it is working as expected. I will get your help in getting a version of the tag checked out.

Thanks again
Peter

Reply:
@lawrencepj1 and @slevisconsulting OK, I went into the history of this to figure this out. It looks like this line IS in place in the main-dev version of CTSM, but removed on the shifting cultivation branch in the update from ctsm1.0.dev055 to ctsm5.1.dev045. I think the removal was accidental so I think we should add it back in. From what I see the original version of shfitingcultivation included this line as well as versions on main-dev, so there isn't a reason to remove it.

It might be best if we all go over this to make sure we all agree, but let me know what you think.

One thing that happened was that there was a different change in SLASH_HARVESTC  that obscured this issue. But, this also might have been what @slevisconsulting needed to change to get balance checks to pass and if so that would be something we'd need to look closer into.

So @slevisconsulting and @lawrencepj1 how should we do this?

Reply:
@lawrencepj1 @ekluzek @slevisconsulting 
decided to put `hrv_xsmrpool_to_atm_patch` back in the code here and rerun the test-suites or rerun the two tests that fail against baselines showing diffs for slash_harvestc.

We expect these tests to now be bfb against baselines.

Also @lawrencepj1 will confirm that the GU_LU and wood harvest are consistent with the lastest code.

Reply:
From what I can tell, these are the tests that I need to rerun:
```
FAIL ERP_P36x2_Lm13.f10_f10_mg37.IHistClm51Bgc.cheyenne_gnu.clm-monthly BASELINE ctsm5.1.dev118: DIFF
FAIL ERP_P36x2_Lm13.f10_f10_mg37.IHistClm51Bgc.cheyenne_intel.clm-monthly BASELINE ctsm5.1.dev118: DIFF
FAIL LCISO_Lm13.f10_f10_mg37.IHistClm51BgcCrop.cheyenne_intel.clm-ciso_monthly BASELINE ctsm5.1.dev118: DIFF
```
Starting them now and will update this post when they complete.

Reply:
All three passed. I will commit my mods and push.